packages/core/src/shared/plugins/suggestion/SuggestionPlugin.ts

```diff
@@ -195,6 +195,17 @@ class SuggestionPluginView<
   getStaticParams(): SuggestionsMenuStaticParams<T> {
     return {
       itemCallback: (item: T) => this.itemCallback(item),
+      getReferenceRect: () => {
+        const decorationNode = document.querySelector(
+          `[data-decoration-id="${this.pluginState.decorationId}"]`
+        );
+
+        if (!decorationNode) {
```

Comment:
is this expected, or should it throw an error?

Reply:
This one is also ok, again if `decorationNode` is `undefined`, it means the element is hidden so the bounding box for it doesn't exist. That means the Tippy is also hidden so it doesn't try to listen for position changes.